README.md

```diff
@@ -1,2 +1,218 @@
-# HelloMicroverse
-Project for module 1 day 2
+<!-- Project for module 1 day 2 -->
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents 
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [HelloMicroverse] <a name="about-project"></a>
+
+> This project is about a excercise to understand Microverse tools
+
+**[Hello Microverse]** is a project to practice whit the Microverse tools
+
+## 🛠 Built With <a name="built-with">Visual Code Editor</a>
+
+### Tech Stack <a name="tech-stack">HTML/CSS</a>
+
+> HTML & CSS is markup metodology to undertand how is the web structured.
+
+<details>
+  <summary>Tool</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Visual Code Editor</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Tool</summary>
+  <ul>
+    <li><a href="https://www.w3.org/standards/">Client PC - Web Browser</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.w3.org/standards/">No Database requiered</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Tool used on Microverse
+
+- **[Visual Studio Code]**
+- **[GitBash]**
+- **[HTML]**
+- **[CSS]**
```

Comment:
As the previous reviewer suggested, kindly update this section to represent a few features of the hello microverse project🙏. Some ideas could be:
- Uses Github actions and linters
- Simple webpage with basic HTML and CSS